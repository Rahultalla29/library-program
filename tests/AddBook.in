ADD BOOK sample1.csv 111111
ADD BOOK sample1.csv 111112
ADD BOOK ThisFileDoesNotExist.csv 111111
ADD BOOK sample1.csv 112681
ADD BOOK sample1.csv 111111
add book sample1.csv 111112
EXIT