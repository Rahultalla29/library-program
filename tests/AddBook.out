user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Hobbit (J.R.R. Tolkien).

user: No such file.

user: No such book in file.

user: Book already exists in system.

user: Book already exists in system.

user: Ending Library process.
